### DISCUSSION TITLE
怎么才能跟官网的编辑器页面一样？

### DISCUSSION THREAD

**Discussion by asas232** [baidu/amis#10509] [General]
我自己的页面一打开就是这个，但有时候要编辑多个页面，又要重置一次，很麻烦
![image](https://github.com/baidu/amis/assets/23547059/edd78d77-d763-4f45-aaaa-190bd5bd420a)
官网的一打开就是这个
![image](https://github.com/baidu/amis/assets/23547059/38da63b6-7a37-45f8-ad91-ecb120607eaa)

**Comment by zzm1122** [comment]
demo示例地址https://github.com/aisuda/amis-editor-demo
